src/scales/categoryScale.ts

```diff
@@ -98,6 +112,22 @@ namespace Plottable.Scales {
     }
 
     /**
+     * Take the scale and drop ticks at regular intervals such that the
+     * resultant ticks are all a reasonable minimum distance apart. Return the
+     * resultant ticks to render, as well as the new stepWidth between them.
+     *
+     * @return {DownsampleInfo} an object holding the resultant domain and new
+     * stepWidth.
+     */
+    public getDownsampleInfo(): DownsampleInfo {
+      const downsampleRatio = Math.ceil(Category._MINIMUM_WIDTH_PER_LABEL_PX / this.stepWidth());
```

Comment:
Oh gridlines component... hmm... okay this is fine. Maybe in the future we should tie gridlines to a specific axis instead of a specific scale?